docs/README.md

```diff
@@ -0,0 +1,26 @@
+Table of Contents
+
+- [Introduction](index.md)
+- [AsyncAPI Specification File](asyncapi-file.md)
+- [Configuration File](configuration-file.md)
+   - [Special parameters](configuration-file.md#special-parameters)
+
+- [Templates](template.md)
+- [File Templates](file-templates.md)
+- [Authoring Templates](authoring-templates.md)
+   - [Common assumptions](authoring-templates.md#common-assumptions)
```

Comment:
Hmm **Common Assumptions** doesn't sound right for a doc's site outline title. 🤔 

This sounds like something we either need to (a) find a diff title; or (b) incorporate this section into another one.

What do you think @pratik2315 @Florence-Njeri ?